Makefile

```diff
@@ -1,5 +1,16 @@
 .PHONY: test test-unit test-integration demo deploy-local linter install build client drand relay-http relay-gossip relay-s3
 
+# Version values
+MAYOR=0
+MINOR=1
+PATCH=0
```

Comment:
how do these get set?
previously this was set from the git tag, it seems now it will need manual updating to the makefile?

Reply:
At this moment, this is being set manually, but we could use something like [this action ](https://github.com/booxmedialtd/ws-action-parse-semver) to parse gh tag and set those values automatically.